src/harness/fourslash.ts

```diff
@@ -2385,7 +2385,10 @@ namespace FourSlash {
             const codeFixes = this.getCodeFixActions(this.activeFile.fileName, errorCode);
 
             if (!codeFixes || codeFixes.length === 0) {
-                this.raiseError("No codefixes returned.");
+                if (expectedTextArray.length !== 0) {
+                    this.raiseError("No codefixes returned.");
+                }
+                return;

```

Comment:
`importNameCodeFixUMDGlobalReact2.ts` asserts that the code fixes are currently empty. Previously this method would have raised an error "No codefixes returned" even if that was exactly what you were asserting.